storefront-reference-architecture-master/cartridges/app_storefront_PD/cartridge/experience/components/commerce_assets/banner.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var Template = require('dw/util/Template');
+var ImageTransformation = require('*/cartridge/experience/utilities/ImageTransformation.js');
+var HashMap = require('dw/util/HashMap');
+
+/**​
+
+ * Render logic for the component.​
+
+ */​
+
+module.exports.render = function (context) {
+
```

Comment:
Watch out for redundant white spaces and consistent formatting

Reply:
fixed it